jmxtrans-core/src/test/java/com/googlecode/jmxtrans/model/PropertyResolverTests.java

```diff
@@ -1,120 +0,0 @@
-/**
- * The MIT License
- * Copyright (c) 2010 JmxTrans team
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-package com.googlecode.jmxtrans.model;
-
-import com.google.common.collect.ImmutableMap;
-import org.assertj.core.api.Assertions;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-import java.util.TreeMap;
-
-import static com.google.common.collect.Maps.newHashMap;
-
-/**
- * @author lanyonm
- */
-public class PropertyResolverTests {
-
-	@Before
-	public void setSomeProperties() {
-		System.setProperty("myhost", "w2");
-		System.setProperty("myport", "1099");
-	}
-
-	@Test
-	public void testProps() {
```

Comment:
Disregard the comment, I just saw that this test has moved to a new package. It's just fine... :+1: